helm/charts/aleph/templates/api.yaml

```diff
@@ -149,6 +149,12 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
+{{- if .Values.api.serviceAnnotations }}
+  annotations:
+  {{- range $key, $value := .Values.api.serviceAnnotations }}
```

Comment:
Can you implement it like helm does it [here](https://github.com/helm/helm/blob/main/pkg/chartutil/create.go#L262C1-L265C13)?